### ISSUE TITLE
Zanite Ring crash

### ISSUE THREAD

**Issue by Volkove** [gildedgames/aether-issues#239]
Hello, did a dungeon and got a Zanite Ring. When I equipped it if I clicked on any block my game crashed. Server posted this error. 

[WARNING] Failed to handle packet for VOLKOVE/192.168.1.1: java.lang.NullPointerException
java.lang.NullPointerException
    at net.aetherteam.aether.player.PlayerAether.getStrengthVsBlock(PlayerAether.java:456)
    at net.aetherteam.aether.player.PlayerEventHandler.onBlockBreak(PlayerEventHandler.java:62)
    at net.minecraftforge.event.ASMEventHandler_13_PlayerEventHandler_onBlockBreak_BreakSpeed.invoke(.dynamic)
    at net.minecraftforge.event.ASMEventHandler.invoke(ASMEventHandler.java:39)
    at net.minecraftforge.event.EventBus.post(EventBus.java:108)
    at net.minecraftforge.event.ForgeEventFactory.getBreakSpeed(ForgeEventFactory.java:42)
    at net.minecraft.entity.player.EntityPlayer.getCurrentPlayerStrVsBlock(EntityPlayer.java:931)
    at net.minecraftforge.common.ForgeHooks.blockStrength(ForgeHooks.java:170)
    at net.minecraft.block.Block.func_71908_a(Block.java:703)
    at net.minecraft.item.ItemInWorldManager.func_73074_a(ItemInWorldManager.java:194)
    at net.minecraft.network.NetServerHandler.func_72510_a(NetServerHandler.java:493)
    at net.minecraft.network.packet.Packet14BlockDig.func_73279_a(SourceFile:46)
    at net.minecraft.network.TcpConnection.func_74428_b(TcpConnection.java:462)
    at net.minecraft.network.NetServerHandler.func_72570_d(NetServerHandler.java:141)
    at net.minecraft.network.NetworkListenThread.func_71747_b(NetworkListenThread.java:54)
    at net.minecraft.server.dedicated.DedicatedServerListenThread.func_71747_b(SourceFile:30)
    at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:691)
    at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:276)
    at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:587)
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:484)
    at net.minecraft.server.ThreadMinecraftServer.run(SourceFile:583)
 [INFO] VOLKOVE left the game

Had full suit of Zanite armor including gloves on. Nothing else. Once I reloaded into the game I un-equipped the ring and had no issues. Re-equipping and clicking a block got same problem. Also was able to re-create error in singleplayer. Zanite pendant does not cause the issue even though both say they have the same effect. 1 other mod on the server is backpack mod from Eydamos.

**Comment by Klappergebiss** [comment]
I've experienced this bug a few days ago. It appeared while playing on the AOS. I had a zanite ring equipped, it worked for lots of blocks as it should work, but when the ring was about to break, it didn't break but crashed minecraft. Took some time to get to know what this crash was caused by. I unequipped the ring and threw it into lava ;P now it doesn't crash anymore. :) 
@Brenwwe so the bad news is, the bug still exists. I'm playing on a mac btw. Maybe it's only fixed on non-Mac Computers...?
